systemtests/tests/python-fd-plugin-postgres-test/python-modules/bareos-fd-local-fileset-with-restoreobjects.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
Also consider to delete this file.

Reply:
You are probably right but this file is not part of the PR?

Reply:
This file is part of this PR, because it appears in the file list.